lib/cartopy/crs.py

```diff
@@ -1251,7 +1251,14 @@ def _rings_to_multi_polygon(self, rings, is_ccw):
             box = sgeom.box(x3, y3, x4, y4, ccw=is_ccw)
 
             # Invert the polygons
-            polygon = box.difference(sgeom.MultiPolygon(interior_polys))
+            try:
+                polygon = box.difference(sgeom.MultiPolygon(interior_polys))
```

Comment:
Is it possible to call `make_valid()` before doing the `box.difference()` call here? Or would that be just as slow potentially...

Reply:
Actually that seems to take no time at all.  Let's do that :+1: